src/services/reviews.js

```diff
@@ -761,9 +761,22 @@ async function handleComments(comments, resourceId, userId) {
 		// Separate comments into ones that need to be updated and ones that need to be created
 		const commentsToUpdate = []
 		const commentsToCreate = []
-
+		let parentCommentCheckFilter = []
```

Comment:
@adithyadinesh0412 we need to validate parentId before creating or updating comment